powerauth-push-server/src/main/java/io/getlime/push/repository/PushCampaignRepository.java

```diff
@@ -20,7 +20,7 @@
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
-import javax.transaction.Transactional;
+import jakarta.transaction.Transactional;
```

Comment:
I have addressed this at https://github.com/wultra/powerauth-push-server/pull/613